dxr/plugins/core.py

```diff
@@ -429,6 +429,11 @@ class FileToIndex(dxr.indexers.FileToIndex):
     def __init__(self, path, contents, plugin_name, tree, vcs):
         super(FileToIndex, self).__init__(path, contents, plugin_name, tree)
         self.vcs = vcs
+        self.extension = splitext(self.path)[1].lower()
```

Comment:
The dropping everything to lowercase doesn't thrill me, because sometimes extensions actually are case-sensitive. The worst of these is .C, which means C++ to gcc.